### ISSUE TITLE
Add a unique counts per column

### ISSUE THREAD

**Issue by chris1610** [chris1610/sidetable#4]
Gave an example in this post - 
```
unique_counts = pd.DataFrame.from_records([(col, df[col].nunique()) for col in df.columns],
                          columns=['Column_Name', 'Num_Unique']).sort_values(by=['Num_Unique'])
```

https://pbpython.com/pandas_dtypes_cat.html

**Comment by chris1610** [comment]
May not be worth it - df.describe(exclude='number').T

Leave open for now and think about it.

**Comment by chris1610** [comment]
Implemented in 0.7
